v_frame/Cargo.toml

```diff
@@ -22,3 +22,4 @@ serde = { version = "1.0", features = ["derive"], optional = true }
 wasm-bindgen = { version = "0.2.63", optional = true }
 rust_hawktracer = "0.7.0"
 new_debug_unreachable = "1.0.4"
+arrayref = "0.3.6"
```

Comment:
Maybe add a note that it will be unneeded from a future version of rust.

Reply:
Following the advice of @tdaede I dropped arrayref altogether and we just use `slice::from_raw_parts` now